chef_master/source/resource_ohai.rst

```diff
@@ -48,114 +47,162 @@ Properties
 
 The ohai resource has the following properties:
 
+``plugin``
+   **Ruby Type:** String
+
+   The name of an Ohai plugin to be reloaded. If this property is not specified, the chef-client will reload all plugins.
+
+.. end_tag
+
+Common Resource Functionality
+=====================================================
+
+Chef resources include common properties, notifications, and resource guards.
+
+Common Properties
+-----------------------------------------------------
+
+.. tag resources_common_properties
+
+The following properties are common to every resource:
+
 ``ignore_failure``
-   **Ruby Type:** true, false | **Default Value:** ``false``
+  **Ruby Type:** true, false | **Default Value:** ``false``
 
-   Continue running a recipe if a resource fails for any reason.
+  Continue running a recipe if a resource fails for any reason.
 
-``notifies``
-   **Ruby Type:** Symbol, 'Chef::Resource[String]'
+``retries``
+  **Ruby Type:** Integer | **Default Value:** ``0``
 
-   .. tag resources_common_notification_notifies
+  The number of times to catch exceptions and retry the resource.
```

Comment:
Consider: "The number of attempts to catch..."

Reply:
Updated